components/Layout/Layout.jsx

```diff
@@ -13,7 +13,7 @@ export default function Layout({ children, Progress, isText }) {
         <meta name="description" content="Emissions calculator" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Flex minHeight="100vh" direction="column">
+      <Flex minHeight="100vh" direction="column" marginBottom={"60px"}>
```

Comment:
the `marginBottom={"60px"}` attribute causes a scrollbar to be ever present on the page, for some reason...
do we need that attribute?

```suggestion
      <Flex minHeight="100vh" direction="column">
```

Reply:
Yeah. In the mobile screens the buttons are jammed up next to the bottom of the screen.
It looks fine there, but there might be better ways to achieve this.